app/models/organizational_unit.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Superclass of Team and User
+class OrganizationalUnit < Sequel::Model
```

Comment:
We could include the `class_table_inheritance` plugin and the `type`/`kind` column already I think.